router/v3/qall.go

```diff
@@ -180,8 +180,7 @@ func (h *Handlers) PatchRoomMetadata(c echo.Context) error {
 
 	livekitClient := lksdk.NewRoomServiceClient(h.Config.LiveKitHost, h.Config.LiveKitAPIKey, h.Config.LiveKitAPISecret)
 	// Find the room
-	var targetRoom *qall.RoomWithParticipants
-	targetRoom = h.QallRepo.GetRoomState(roomID.String())
+	var targetRoom *qall.RoomWithParticipants = h.QallRepo.GetRoomState(roomID.String())
```

Comment:
すごく細かいのでこれは気になればで全然構いませんが、この書き方でも大丈夫だとは思います…！
```suggestion
	targetRoom := h.QallRepo.GetRoomState(roomID.String())
```